Userland/Applications/FileManager/PropertiesWindow.cpp

```diff
@@ -26,46 +27,57 @@
 #include <string.h>
 #include <unistd.h>
 
-PropertiesWindow::PropertiesWindow(DeprecatedString const& path, bool disable_rename, Window* parent_window)
+ErrorOr<NonnullRefPtr<PropertiesWindow>> PropertiesWindow::try_create(DeprecatedString const& path, bool disable_rename, Window* parent)
+{
+    auto window = TRY(adopt_nonnull_ref_or_enomem(new (nothrow) PropertiesWindow(path, parent)));
+    window->set_icon(TRY(Gfx::Bitmap::load_from_file("/res/icons/16x16/properties.png"sv)));
+    TRY(window->create_widgets(disable_rename));
+    return window;
+}
+
+PropertiesWindow::PropertiesWindow(DeprecatedString const& path, Window* parent_window)
     : Window(parent_window)
 {
     auto lexical_path = LexicalPath(path);
 
-    auto main_widget = set_main_widget<GUI::Widget>().release_value_but_fixme_should_propagate_errors();
-    main_widget->set_layout<GUI::VerticalBoxLayout>();
-    main_widget->layout()->set_spacing(6);
-    main_widget->layout()->set_margins(4);
-    main_widget->set_fill_with_background_color(true);
+    m_name = lexical_path.basename();
+    m_path = lexical_path.string();
+    m_parent_path = lexical_path.dirname();
 
     set_rect({ 0, 0, 360, 420 });
     set_resizable(false);
+}
 
-    set_icon(Gfx::Bitmap::load_from_file("/res/icons/16x16/properties.png"sv).release_value_but_fixme_should_propagate_errors());
-
-    auto& tab_widget = main_widget->add<GUI::TabWidget>();
+ErrorOr<void> PropertiesWindow::create_widgets(bool disable_rename)
+{
+    auto main_widget = TRY(set_main_widget<GUI::Widget>());
+    main_widget->set_layout<GUI::VerticalBoxLayout>();
```

Comment:
You missed a single `try_set_layout()` :wink:

Reply:
Oops, well spotted! Should be fixed now :^)